File: problemas/matematica/2497/2497.swift
import Foundation

var teste = 1
while true {
    let n = Int(readLine(strippingNewline: true)!)!
    if (n == -1) {
        break
    }

    print(String(format: "Experiment %d: %d full cycle(s)", teste, n / 2))
    teste += 1
}

File: problemas/matematica/2497/2497.go
package main

import (
	"fmt"
)

func main() {
	var n, teste int

	teste = 1
	for {
		fmt.Scanf("%d\n", &n)
		if (n == -1) {
			break
		}

		fmt.Printf("Experiment %d: %d full cycle(s)\n", teste, n / 2)
		teste += 1
	}
}
